resources/testdriver.js

```diff
@@ -170,6 +170,22 @@
          */
         freeze: function() {
             return window.test_driver_internal.freeze();
+        },
+
+        /**
+         * Send a sequence of pointer actions
+         *
+         * This function sends a sequence of pointer actions to the top level window
+         * to perform. It is modeled after the behaviour of {@link
+         * https://w3c.github.io/webdriver/#actions|WebDriver Actions Command}
+         *
+         * @param {Array} actions - an array of action objects to perform as
+         *                          described in docs
+         * @returns {Promise} fufiled after the actions are performed, or rejected in
+         *                    the cases the WebDriver command errors
+         */
+        pointer_action_sequence(actions) {
+            return window.test_driver_internal.pointer_action_sequence(actions);
```

Comment:
Do we want to do similar logic to click that we did before like calculating the x,y from the given element with the same logic or scrollIntoView here?

Reply:
Yep, I think if the input is an element we need to do that conversion here. Good catch!

Reply:
Given a bit more thought, I think I want to discuss this a bit more.
What do you think about the providing coordinates vs. providing an element for the API surface.

One approach would be to keep the surface as it is now and infer whether an Element or an object with x, y attributes is being passed in and react accordingly.

Another is to have it always be x, y coordinates and maybe providing a util function (that could live in testdriver?) to calculate these given an element for ease of use.

WYT?

Reply:
I like the idea of having x, y only. I just wanted to follow the click API and also support the element thing but if we can keep the API surface simple and have util functions as you suggested IMO it is clearer.

Reply:
That sounds good to me, I will push a change :)